service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -59,6 +65,8 @@ class DatasetServiceTest {
   private final AuthenticatedUserRequest user = mock(AuthenticatedUserRequest.class);
   private final DatasetId datasetId = new DatasetId(UUID.randomUUID());
   private final String sourceId = "sourceId";
+
+  private final String externalSourceId = "phs000000.v11.p8";
```

Comment:
I think in practice external source IDs will be UUIDs minted either within catalog or by catalog's callers. But for this test it shouldn't matter what the source ID is. Is there a reason why `DatasetServiceTest.sourceId` wouldn't work?

Reply:
Since I created a mock external dataset, I also created a mock `externalSourceId`. The regular `sourceId` could work as long as the mock external dataset also contained that `sourceId`. It would be confusing to me if I was looking in the debugger and I saw a mock external dataset with just the  regular `sourceId`, though (in fact this happened to me initially).

Reply:
I ended up removing this due to removing the test for external datasets.